web/pom.xml

```diff
@@ -329,50 +295,6 @@
     <build>
         <finalName>app</finalName>
         <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-dependency-plugin</artifactId>
-                <version>2.3</version>
-                <executions>
-                    <execution>
-                        <id>unpack</id>
-                        <phase>generate-resources</phase>
-                        <goals>
-                            <goal>unpack</goal>
-                        </goals>
-                        <configuration>
-                            <artifactItems>
-                                <artifactItem>
-                                    <groupId>${frontend.groupId}</groupId>
-                                    <artifactId>${frontend.artifactId}</artifactId>
-                                    <version>${frontend.version}</version>
-                                    <type>jar</type>
-                                    <outputDirectory>dist</outputDirectory>
-                                    <overWrite>true</overWrite>
-                                    <excludes>META-INF/</excludes>
-                                </artifactItem>
-                            </artifactItems>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>
-            <plugin>
-                <artifactId>maven-war-plugin</artifactId>
-                <version>2.3</version>
-                <configuration>
-                    <webResources>
-                        <resource>
-                            <directory>dist</directory>
-                            <targetPath/>
-                        </resource>
-                    </webResources>
-                    <archive>
-                        <manifest>
-                            <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
-                        </manifest>
-                    </archive>
-                </configuration>
-            </plugin>
```

Comment:
Does this build out the html for the old frontends? Will that impact the swagger pages at all?